src/installer/pkg/projects/Directory.Build.targets

```diff
@@ -319,12 +319,42 @@
       Condition="'%(FrameworkListRootAttributes.Value)' == ''"
       Text="Missing value for property 'FrameworkList%(FrameworkListRootAttributes.Identity)'" />
 
+    <ItemGroup>
+      <!-- UNIX -->
+      <SingleFileHostInclude Include="libSystem.Globalization.Native.so"/>
```

Comment:
Call this item list `SingleFileHostIncludeFilename` instead to make the naming more straightforward?